.travis.yml

```diff
@@ -10,5 +10,6 @@ script:
 branches:
     only:
       - master
+      - will/upgrade-django-rest-framework
```

Comment:
no. currently tests on travis run against master branch so this is needed to run the build.